pkg/controller/compliancescan/scan.go

```diff
@@ -134,7 +134,7 @@ func newScanPodForNode(scanInstance *compv1alpha1.ComplianceScan, node *corev1.N
 					Command: []string{
 						"sh",
 						"-c",
-						fmt.Sprintf("rm -rf %s && mkdir -p %s && ln -s %s %s | /bin/true", KubeletConfigLinkPath, KubeletConfigLinkFolder, KubeletConfigMapPath, KubeletConfigLinkPath),
+						fmt.Sprintf("mkdir -p %s && ln -s %s %s | /bin/true", KubeletConfigLinkFolder, KubeletConfigMapPath, KubeletConfigLinkPath),
```

Comment:
I wonder if we can stand up a test case or suite that hammers on a cluster with a bunch of profiles across products in the same binding to ensure they don't come back as inconsistent?